scripts/error_codes.py

```diff
@@ -221,7 +221,7 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
 
     old_source_only_ids = {
         "1123", "1220", "1584", "1823", "1950",
-        "1988", "2657", "2800",
+        "1988", "2066", "2657", "2800",
```

Comment:
Not sure yet how to add tests, commandline?

Reply:
Either commandline or standardJSON via the commandLineTests.sh should work I guess

Reply:
Added a cmdlineTest, but apparently that is ignored by `error_codes.py`.

Reply:
How so? Tests passed

Reply:
Ah, the error id is still here..